tests/main.tf

```diff
@@ -0,0 +1,123 @@
+module "lx-instance-centos6" {
+  source = "../modules/lx-instance"
+
+  Name             = "${local.test_prefix}-lx-instance-centos6"
+  AmiId            = "${data.aws_ami.centos6.image_id}"
+  AmiDistro        = "CentOS"
+  AppVolumeSize    = "10"
```

Comment:
I'm thinking we can pass the url to the cloudwatch agent as well. That _might_ help capture logs we can evaluate in case things start breaking...

Reply:
added